### PR TITLE
Add basic arguments validation

### PR DESCRIPTION
**Pull request by Herve-M**
This PR is the first step of validation of argument(s) passed through OpenRA.Utility. Target issue #8442
- Add a method [IUtilityCommand](https://github.com/Herve-M/OpenRA/blob/feat-utility-polish/OpenRA.Game/IUtilityCommand.cs#L20)
- Test Argument before calling command [OpenRA.Utility/Program.cs](https://github.com/Herve-M/OpenRA/blob/feat-utility-polish/OpenRA.Utility/Program.cs#L59-L72)
- Impl. basic validation (on args.Length) on all [Utility.Commands](https://github.com/Herve-M/OpenRA/commit/ff82276c2103ab2c06e5926f37170346a7ee8795?diff=unified)

If a command is called without validating, the description is showed.

```
λ OpenRA.Utility.exe ra --shp
Invalid arguments for '--shp'
  --shp PNGFILE [PNGFILE ...]
  Combine a list of PNG images into a SHP
```

If a inexisting command is used, a System.ArgumentException is thrown without crashing / showing it. 

```
λ OpenRA.Utility.exe ra --sh
No such command '--sh'
```

**Special attention for validation** : see if UnityCommands:ValidateArguments are just.

### CODE REVIEW

File: OpenRA.Mods.Common/UtilityCommands/CheckCodeStyle.cs [hunk 1]
```diff
@@ -24,6 +24,16 @@ class CheckCodeStyle : IUtilityCommand
...
 		int violationCount;
 
+		public bool ValidateArguments(string[] args)
+		{
+			if (args.Length >= 2)
+			{
+				return true;
+			}
+
+			return false;
```
Comment (Phrohdoh) [inline OpenRA.Mods.Common/UtilityCommands/CheckCodeStyle.cs]: `return args.Length >= 2;`

**Comment by RoosterDragon** [comment]
Looks good generally.

- Can you squash your fixes into your original commit please.
- If invalid arguments are given to a command, can you print out a message saying something like `Invalid arguments for '--shp'` and then print the usage to make it clear there was a problem.
- If the command doesn't exist, rather than crashing or showing an exception, I think just outputting an error message (e.g. `No such command '--sh'`) would be sufficient.

**Comment by Herve-M** [comment]
@RoosterDragon All done.

**Comment by penev92** [comment]
Looks good to me :+1:

**Comment by obrakmann** [comment]
:+1: Works and looks fine.

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/dae2dfc646f525cbd4ba66e4f79a418c7fa3e46f)

**Pull request merged**